GrafanaDatastoreServer.py

```diff
@@ -66,8 +68,10 @@ def query():
     # !!! dates 'from' and 'to' are expected to be in UTC, which is what Grafana provides here.
     # If not in UTC, use pytz to set to UTC timezone and subtract the utcoffset().
     # Time delta calculations should always be done in UTC to avoid pitfalls of daylight offset changes.
-    stime = (dateutil.parser.parse(request['range']['from']) - EPOCH) / timedelta(milliseconds=1)
-    etime = (dateutil.parser.parse(request['range']['to']) - EPOCH) / timedelta(milliseconds=1)
+    stime = (dateutil.parser.parse(request['range']['from']) - EPOCH).total_seconds() * 1000
```

Comment:
@maguec we'll get the same error in case `dateutil.parser.parse(request['range']['from'])` is without timezone. I think we should check in runtime and use the right calculation accordingly